### DISCUSSION TITLE
sh autogen.sh error

### DISCUSSION THREAD

**Discussion by skoowoo** [protobuf-c/protobuf-c#670] [Q&A]
Linux ubuntu 3.8.0-29-generic

autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4 ${ACLOCAL_FLAGS}
autoreconf: configure.ac: tracing
autoreconf: configure.ac: creating directory build-aux
autoreconf: running: libtoolize --install --copy --force
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR,`build-aux'.
libtoolize: can not copy `/usr/share/libtool/config/config.guess' to`build-aux/'
libtoolize: can not copy `/usr/share/libtool/config/config.sub' to`build-aux/'
libtoolize: can not copy `/usr/share/libtool/config/install-sh' to`build-aux/'
libtoolize: can not copy `/usr/share/libtool/config/ltmain.sh' to`build-aux/'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: can not copy`/usr/share/aclocal/libtool.m4' to `m4/'
libtoolize: can not copy`/usr/share/aclocal/ltoptions.m4' to `m4/'
libtoolize: can not copy`/usr/share/aclocal/ltsugar.m4' to `m4/'
libtoolize: can not copy`/usr/share/aclocal/ltversion.m4' to `m4/'
libtoolize: can not copy`/usr/share/aclocal/lt~obsolete.m4' to `m4/'
autoreconf: libtoolize failed with exit status: 1

**Comment by edmonds** [comment]
I assume this is Ubuntu 12.04 LTS?

I cannot reproduce this build failure on that platform, can you please provide more details about your environment?
